README.md

```diff
@@ -7,3 +7,36 @@
 ```
 mvn clean install
 ```
+
+## Usage
+
+### Dependency Check (OWASP)
```

Comment:
Starting a thread to answer https://github.com/connexta/ion-ingest-api/pull/9#issuecomment-503807050

- I think it's fine to keep generating all of those clients and keep the stub server. We can remove them later if they end up being too much trouble to maintain.
- We don't have many suppressions right now, so I don't think we need a solution to share suppressions across all of the ion-xxx-api repos yet.

Reply:
@emmberk, my only concern with generating clients is that the team that owns the repo is also responsible for keeping those clients up-to-date moving forward, including making sure they remain tested and backward compatible. So this should probably be a conscious decision and not just something we teams always do blindly.
And there's also the question of which client(s) should be generated. Should it be up to the team that owns the API to decide? And how about the server stub(s)?

Reply:
I don't have a strong opinion about what we should do going forward. I _do_ think that it's not an issue we need to prioritize right now seeing we don't have any tests at all. The only trouble that we have right now because of these extra generated things is maintaining the OWASP check, which isn't that much of a burden right now.